.github/workflows/ci-cd.yml

```
@@ -0,0 +1,114 @@
+---
+name: CI/CD
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  workflow_dispatch:
+
+env:
+  FORCE_COLOR: "1" # Make tools pretty.
+  PIP_DISABLE_PIP_VERSION_CHECK: "1"
+  PIP_NO_PYTHON_VERSION_WARNING: "1"
+  PYTHON_LATEST: "3.11"
+  SETUPTOOLS_SCM_PRETEND_VERSION: "1.0" # avoid warnings about shallow checkout
+
+  # For re-actors/checkout-python-sdist
+  sdist-artifact: python-package-distributions
+  sdist-name: aiomonitor-1.0.tar.gz
+
+permissions:
+  contents: read
+
+jobs:
+  build-sdist:
+    name: 📦 Build the source distribution
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.PYTHON_LATEST }}
+      - run: python -Im pip install build
+
+      - run: python -Im build --sdist
```

Build both. When there's no args, the wheel will be created from sdist and not Git checkout. So it's also a built-in smoke test.
```suggestion
      - run: python -Im build
```